docs/man_pages/project/configuration/edit-configuration.md

```diff
@@ -34,4 +34,6 @@ Command | Description
 [prop add](prop-add.html) | Enables more options for the selected project property, if the property accepts multiple values.
 [prop remove](prop-remove.html) | Disables options for the selected project property, if the property accepts multiple values.
 [prop set](prop-set.html) | Sets the selected project property and overwrites its current value.
+[web-view](web-view.html) | Lists all supported web views for different mobile platforms.
```

Comment:
Lists the available web views for iOS and Android.

Note: Does this work for Windows Phone?